public/apps/configuration/panels/user-list.tsx

```diff
@@ -55,42 +57,54 @@ function dictView() {
   };
 }
 
-const columns = [
-  {
-    field: 'username',
-    name: 'Username',
-    render: (text: string) => (
-      <a href={buildHashUrl(ResourceType.users, Action.view, text)}>{text}</a>
-    ),
-    sortable: true,
-  },
-  {
-    field: 'attributes',
-    name: 'Attributes',
-    render: dictView(),
-    truncateText: true,
-  },
-];
+function getColumns(currentUsername: string) {
+  return [
+    {
+      field: 'username',
+      name: 'Username',
+      render: (username: string) => (
+        <>
+          <a href={buildHashUrl(ResourceType.users, Action.view, username)}>{username}</a>
+          {username === currentUsername && (
+            <>
+              &nbsp;
+              <EuiBadge>Current</EuiBadge>
+            </>
+          )}
+        </>
+      ),
+      sortable: true,
+    },
+    {
+      field: 'attributes',
+      name: 'Attributes',
+      render: dictView(),
+      truncateText: true,
+    },
+  ];
+}
 
 export function UserList(props: AppDependencies) {
   const [userData, setUserData] = useState<InternalUsersListing[]>([]);
   const [errorFlag, setErrorFlag] = useState(false);
   const [selection, setSelection] = useState<InternalUsersListing[]>([]);
   const [isActionsPopoverOpen, setActionsPopoverOpen] = useState(false);
   const [currentUsername, setCurrentUsername] = useState('');
-  // TODO: fetch current username
+
   useEffect(() => {
     const fetchData = async () => {
       try {
-        setUserData(await getUserList(props.coreStart.http));
+        const userDataPromise = getUserList(props.coreStart.http);
+        setCurrentUsername((await getAuthInfo(props.coreStart.http)).user_name);
+        setUserData(await userDataPromise);
```

Comment:
Minor: line 97 can be merged into line 99 just as your original code. Any reason for the separation?

Reply:
Trying to make 2 requests in parallel... though don't know how much time can be saved...

Reply:
Sure. Just want to make sure that it is intentional.